src/domain/about/AboutPage.js

```diff
@@ -0,0 +1,55 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { FormattedHTMLMessage } from 'react-intl';
+
+import AboutPartners from './AboutPartners';
+import PageWrapper from '../../../app/pages/PageWrapper';
+import FeedbackLink from '../../../app/shared/feedback-link/FeedbackLink';
+import injectT from '../../../app/i18n/injectT';
+import { getCurrentCustomization } from '../../../app/utils/customizationUtils';
+
+function AboutPage({ t }) {
+  const city = getCurrentCustomization() ? getCurrentCustomization().toLowerCase() : 'default';
+  // TODO: Remove me along with getCurrentCustomization stuff.
```

Comment:
I don't understand this todo, is there something planned that would change current implementation?

Reply:
I leave it as a reminder so i will clean the util function later. Just refactor purpose